frontend/workflows/k8s/src/cordon-node.tsx

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import type { clutch as IClutch } from "@clutch-sh/api";
+import {
+  Button,
+  ButtonGroup,
+  client,
+  Confirmation,
+  MetadataTable,
+  Resolver,
+  useWizardContext,
+} from "@clutch-sh/core";
+import { useDataLayout } from "@clutch-sh/data-layout";
+import type { WizardChild } from "@clutch-sh/wizard";
+import { Wizard, WizardStep } from "@clutch-sh/wizard";
+import { bool } from "yup";
+
+import type { ConfirmChild, ResolverChild, WorkflowProps } from ".";
+
+const NodeIdentifier: React.FC<ResolverChild> = ({ resolverType }) => {
+  const { onSubmit } = useWizardContext();
+  const resolvedResourceData = useDataLayout("resourceData");
+  const resolverInput = useDataLayout("resolverInput");
+
+  const onResolve = ({ results, input }) => {
+    // Decide how to process results.
+    resolvedResourceData.assign(results[0]);
+    resolverInput.assign(input);
+    onSubmit();
+  };
+
+  return <Resolver type={resolverType} searchLimit={1} onResolve={onResolve} />;
+};
+
+const NodeDetails: React.FC<WizardChild> = () => {
+  const { onSubmit, onBack } = useWizardContext();
+  const resourceData = useDataLayout("resourceData");
+  const node = resourceData.displayValue() as IClutch.k8s.v1.Node;
+  const update = (key: string, value: any) => {
+    resourceData.updateData(key, value === "true");
+  };
+  return (
+    <WizardStep error={resourceData.error} isLoading={resourceData.isLoading}>
+      <strong>Node Details</strong>
+      <MetadataTable
+        onUpdate={update}
+        data={[
+          { name: "Name", value: node.name },
+          { name: "Cluster", value: node.cluster },
+          {
+            name: "Unschedulable",
+            value: node.unschedulable,
+            input: {
+              type: "string",
+              key: "unschedulable",
+              validation: bool(),
+            },
```

Comment:
I think it might be better to use a toggle here for the state. Something like
```tsx
value: (<Switch checked={resourceData.unschedulable} onChange={update} />),
```

Reply:
👍 will change it to a toggle